lib/form/js/validation.js

```diff
@@ -2,7 +2,7 @@
 /* global allPlugins, allInputs, allSettings, allIDs */
 
 // TODO: Add In-Browser Tests
-(function formValidation() {
+var validate = (function formValidation() {
```

Comment:
Instead of setting the whole function to a variable, export the function from the closure by setting `window.validate` to the function you want to export